src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
+import classNames from "classnames";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useAdminVenues } from "hooks/useAdminVenues";
+
+import { Venue_v2 } from "types/venues";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueWizard from "pages/Admin/Venue/VenueWizard/VenueWizard";
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import "./AdminAdvancedSettings.scss";
+
+export enum AdminAdvancedTab {
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+}
+
+const adminVenueTabs: Record<AdminAdvancedTab, String> = {
+  [AdminAdvancedTab.basicInfo]: "Start",
+  [AdminAdvancedTab.entranceExperience]: "Entrance",
+  [AdminAdvancedTab.advancedMapSettings]: "Advanced",
+};
+
+const DEFAULT_TAB = AdminAdvancedTab.basicInfo;
+
+export const AdminAdvancedSettings: React.FC = () => {
+  const { user } = useUser();
+  useAdminVenues(user?.uid);
+
+  const venueId = useVenueId();
+  const [selectedTab, setSelectedTab] = useState<string>(DEFAULT_TAB);
+
+  const selectDefaultTab = useCallback(() => {
+    setSelectedTab(DEFAULT_TAB);
+  }, []);
+
+  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
+
+  const renderAdminAdvancedTabs = useMemo(() => {
+    return Object.entries(adminVenueTabs).map(([id, text]) => (
+      <Nav.Link
+        key={id}
+        className={classNames("AdminVenueView__tab", {
+          "AdminVenueView__tab--selected": selectedTab === id,
+        })}
+        eventKey={id}
+      >
```

Comment:
`key={id}` vs `key={key}`
or
`selectedTab === id` vs `selectedTab === tab`
or
`eventKey={id}` vs `eventKey={key}`

Any objective preference for `id` over the rest?

Reply:
> Any objective preference for id over the rest?

Not really. `key` makes more sense than `id`. I will change it.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/b2285b9a5ca9ba5346f299db9386d0571fb1e9d3